apps/communications/dialer/test/marionette/lib/views/keypad/views.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+/* global module */
+var KeypadAccessor = require('./accessors');
+
+function KeypadView(client) {
+  
+  this.client = client;
+  this.accessors = new KeypadAccessor(client);
+
+}
+
+KeypadView.prototype = {
+
+  key: function(keyNumber) {
+    return this.client.helper.waitForElement
+      (this.accessors.getKeypadKeySelector(keyNumber));
+  },
+  
+  typePhoneNumber: function(number) {
+    var i;
+    var typeNumberArray = String(number).split('');
+
+    for (i in typeNumberArray){
+      if(i == '+'){
+        var zero_key = this.key(0);
+        this.actions.longPress(zero_key, 1).perform();
+      }
+      else{
+        i = parseInt(typeNumberArray[i]);
+        this.key(i).tap();
+      }
+    }
+    var keypadView = new KeypadView(this.client);
```

Comment:
Following up from [this comment](https://github.com/mozilla-b2g/gaia/pull/33913#discussion_r50684170).

How about the following?
`keypad_test.js`
```js 
subject.keypadView.typePhoneNumber(12345);
var addToContactsActivities = subject.keypadView.addToContacts();
```

`keypad/view.js`
```js
  typePhoneNumber: function(number) {
    var i;
    var typeNumberArray = String(number).split('');

    for (i in typeNumberArray){
      if(i == '+'){
        var zero_key = this.key(0);
        this.actions.longPress(zero_key, 1).perform();
      }
      else{
        i = parseInt(typeNumberArray[i]);
        this.key(i).tap();
      }
    }
  },
```